### DISCUSSION TITLE
[#991918] "Actors" should be called "entities"

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#39] [Bugs]
The term "actor" is specific to Bedrock's internals, and is not supposed to appear in-game. The correct term is "entity."
![image](https://user-images.githubusercontent.com/26075577/223876262-05501a67-3ccc-4025-8938-d09ad19bfd2b.png)

**Comment by MinecraftMoments** [comment]
i think actors makes sense for the development team, especially if it's relating to coding itself, but for a common user like me, i did not even realize the actors button was in reference to entities, so changing it or adding clarification somewhere would be helpful

**Comment by minecraftchloe** [comment]
We can definitely evaluate changing the name of this!

**Reply by tryashtar** [reply]
It's a low prio nitpick, but I think there's a policy or something for it. The "actor properties" feature got renamed to just "properties" as the final change before it left experimental, and there are a couple other examples. But I can't peek behind the curtain so can't say for sure

**Comment by tryashtar** [comment]
Addressed in today's changelog, thank you
